src/Components/Blazor/WebAssembly.Authentication/src/Options/DefaultApiAuthorizationOptionsConfiguration.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    internal class DefaultApiAuthorizationOptionsConfiguration : IPostConfigureOptions<RemoteAuthenticationOptions<ApiAuthorizationProviderOptions>>
+    {
+        private readonly string _applicationName;
+
+        public DefaultApiAuthorizationOptionsConfiguration(string applicationName) => _applicationName = applicationName;
+
+        public void Configure(RemoteAuthenticationOptions<ApiAuthorizationProviderOptions> options)
+        {
+            options.ProviderOptions.ConfigurationEndpoint ??= $"_configuration/{_applicationName}";
```

Comment:
This URL seems excessively general. Could we add something to the default URL that clarifies this is for *authentication* configuration?

Also, who provides the server-side implementation for this endpoint?

 * If it's in user code, the URL probably shouldn't have this frameworky-underscore naming pattern
 * If it's in framework code, perhaps consider putting it in the `_framework` portion of the URL space so that there isn't another pattern for people to know has magic behaviors. Example: `_framework/authconfiguration?application=name`

Reply:
Let's discuss this more.

Reply:
Discussed offline